aws/synthetix/prod/geth/ecs-params.yml

```diff
@@ -0,0 +1,34 @@
+version: 1
+task_definition:
+  services:
+    geth_l2:
+      essential: true
+#      healthcheck:
+#        test: ["CMD-SHELL", "curl -f -H \"Content-Type: application/json\" -d '{\"jsonrpc\": \"2.0\", \"id\": 9999999, \"method\": \"net_version\"}' http://localhost:9545/ || exit 1"]
+#        interval: 10s
+#        timeout: 5s
+#        retries: 3
+#        start_period: 10s
```

Comment:
What's this used for?

Reply:
It's not currently. Was to be a health check, but the container has successfully stopped and restarted anytime there's a issue with geth, so it's not needed.